jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -21,11 +22,11 @@
                 - culprits
     scm:
         - git:
-            url: https://github.com/kubernetes/kubernetes
+            url: '{url}'
```

Comment:
Our configs won't work on other Jenkins instances without modification anyway. I'd rather keep them as simple as possible so that making the required changes is clear and easy.